### PR TITLE
cli: Polish `assign` and `unassign` commands

### PR DESCRIPTION
**Pull request by cloudhead**
Just some code and ux improvements.

### CODE REVIEW

File: radicle-cli/src/commands/assign.rs [hunk 1]
```diff
@@ -5,58 +5,58 @@ use anyhow::anyhow;
...
+    --to <did>    Assignee to add to the issue
+    --help        Print help
 "#,
 };
 
 #[derive(Debug)]
 pub struct Options {
     pub id: issue::IssueId,
-    pub peer: Did,
+    pub to: Did,
```
Comment (FintanH) [inline radicle-cli/src/commands/assign.rs:32]: Should we allow `Vec<Did>` while we're at it?

File: radicle-cli/src/commands/unassign.rs [hunk 2]
```diff
@@ -12,23 +12,24 @@ use radicle::storage::WriteStorage;
...
+    --from <did>     Assignee to remove from the issue
+    --help           Print help
 "#,
 };
 
 #[derive(Debug)]
 pub struct Options {
     pub id: issue::IssueId,
-    pub peer: Did,
+    pub from: Did,
```
Comment (FintanH) [inline radicle-cli/src/commands/unassign.rs:32]: Same here

**Review by FintanH — APPROVED** [review]
Looks good but suggesting that we allow for multiple `Did`s

File: radicle-cli/src/commands/assign.rs (hunk 1 above)
Comment (cloudhead) [reply radicle-cli/src/commands/assign.rs:32]: Actually I was thinking multiple issues would be more useful to start, eg. assigning someone to multiple issues: `rad assign id1 id2 id3 --to finto`

Comment (cloudhead) [reply radicle-cli/src/commands/assign.rs:32]: As usually we only have one person assigned per issue. But yeah, at the COB level what you proposed is "free".

Comment (FintanH) [reply radicle-cli/src/commands/assign.rs:32]: Ya, I presumed the assigning can have multiple people since the code was using `[]`

**Review by FintanH — APPROVED** [review]
:kissing_cat: :ok_hand:

**Pull request merged**